### PR TITLE
Updates install script

### PR DESCRIPTION
**Pull request by antonmolodykh**

### CODE REVIEW

File: bin/install.sh [hunk 1]
```diff
@@ -82,8 +92,6 @@ sudo find /usr/share/locale -mindepth 1 -maxdepth 1 ! -name 'en*' ! -name 'de*'
 
 cd ~/screenly && git rev-parse HEAD > ~/.screenly/latest_screenly_sha
 
-if test -f ~/.screenly/wifi_set; then  rm ~/.screenly/wifi_set; fi
```
Comment (vpetersson) [inline bin/install.sh]: Why is this removed?

File: bin/install.sh [hunk 2]
```diff
@@ -65,7 +74,8 @@ sudo apt-get update
 sudo apt-get purge -y python-setuptools python-pip python-pyasn1
 sudo apt-get install -y python-dev git-core libffi-dev libssl-dev
 curl -s https://bootstrap.pypa.io/get-pip.py | sudo python
-sudo pip install ansible==2.1.0.0
+
+sudo pip install ansible==2.5.3
```
Comment (vpetersson) [inline bin/install.sh]: This change might break things.

File: bin/install.sh (hunk 1 above)
Comment (antonmolodykh) [reply bin/install.sh]: because we stopped using this (now we are using `initialized` file)

File: bin/install.sh (hunk 2 above)
Comment (antonmolodykh) [reply bin/install.sh]: hmm, I just copied this from the experimental branch. there it does not break compatibility

Comment (antonmolodykh) [reply bin/install.sh]: it seems you have already updated ansible in the requirements file
https://github.com/Screenly/screenly-ose/blob/master/requirements.txt#L3
